### PR TITLE
Refactor header

### PR DESCRIPTION
**Pull request by avoinea**
https://taskman.eionet.europa.eu/issues/147798
https://taskman.eionet.europa.eu/issues/147802

https://62713c57bcbdf4004a3d99b8-layzuefsti.chromatic.com/?path=/story/layout-header--default

### CODE REVIEW

File: theme/themes/eea/extras/header.variables [hunk 1]
```diff
@@ -1,143 +1,149 @@
...
+@computerMegaMenuMarginBottom   : 0.6em;
+
+// Font sizes and line heights
+@megaMenuFontSize: ~"min(max(1.125rem, 3.5vw), 2.2rem)";
+
+/* Fallback font sizes for ie and opera mini */
+@mobileMegaMenuFontSize         : 1.125rem;
+@tabletMegaMenuFontSize         : @h1;
+
+////////@computerMegaMenuFirstFontSize  : 3rem;
```
Comment (ichim-david) [inline theme/themes/eea/extras/header.variables:133]: @g-stamatis  Cleanup this pull request if you still have WIP work

File: src/ui/Header/HeaderSearchPopUp.js [hunk 2]
```diff
@@ -14,16 +14,16 @@ function HeaderSearchPopUp({ onClose, triggerRefs = [] }) {
         <div className="wrapper">
           <Input
             className="search"
-            icon="search"
+            icon={{ className: 'ri-search-line', link: true }}
             placeholder="Search..."
             fluid
           />
-          <div className="action">
+          {/* <div className="action">
```
Comment (ichim-david) [inline src/ui/Header/HeaderSearchPopUp.js:21]: @g-stamatis  why is this commented?

**Comment by avoinea** [comment]
@ichim-david @nileshgulia1 I don't remember what is with this PR. Is it still a valid PR, or should we close it and remove the branch?

**Comment by nileshgulia1** [comment]
@avoinea I see the changes related to Header already in master somehow. Though there are some changes with Banner in this PR 
https://github.com/eea/volto-eea-design-system/pull/114/files#diff-b974272cfcb28f3516a678a05237844ede0b990d5bba6dc302d0c201a8a67080
-cc @ichim-david

**Comment by avoinea** [comment]
@nileshgulia1 Can you please extract the missed changes within a new branch and PR? If they are still needed.

**Comment by ichim-david** [comment]
@eea-jenkins build all

**Comment by g-stamatis** [comment]
@eea-jenkins build all

**Comment by g-stamatis** [comment]
Position stick and transparency for the main section of the header.
Language list included in this branch.

**Review by g-stamatis — APPROVED** [review]

**Comment by ichim-david** [comment]
@eea-jenkins build storybook

**Comment by ichim-david** [comment]
@g-stamatis there is an accessibility issue to the language option
accessibility-warning
I also think the dropdown looks a bit strange when it's opened and you see some of the green bg, it looks like a border
dropdown-opened

**Comment by g-stamatis** [comment]
@eea-jenkins build storybook

**Comment by g-stamatis** [comment]
@eea-jenkins build storybook

**Comment by ichim-david** [comment]
@eea-jenkins build storybook

**Review by ichim-david — APPROVED** [review]
There are still some things to fix but the mega menu and header is already in a much better state that we had until now.
Further fixes will be made in a separate pull request.
Thank you Stam and Antonis

**Pull request merged**